src/decoder/decodeqrdata.ts

```diff
@@ -219,7 +219,7 @@ function decodeByteSegment(bits: BitStream, result: resultByteArray, count: numb
   for (var i = 0; i < count; i++) {
     readBytes[i] = bits.readBits(8);
   }
-  result.val = readBytes
+  Array.prototype.push.apply(result.val, readBytes);
```

Comment:
I didn't want to concat because the array instance changes (and I wasn't sure if thats a problem)
But if you say it's ok i'll change it quickly!